### PR TITLE
Add security config

### PR DESCRIPTION
**Pull request by hahava**
- Allowing request with start "/api/*"

### CODE REVIEW

**Comment by imbyungjun** [comment]
Please add authorization configuration.
You know... admin, user and so on...

**Comment by hahava** [comment]
> Please add authorization configuration.
> You know... admin, user and so on...

I added basic auth. Please approve pull request

File: backend/src/main/kotlin/org/dontstw/eventfactory/domain/Member.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.dontstw.eventfactory.domain
+
+data class Member(
+        val uId: String,
+        var uPw: String,
+        var uName: String
```
Comment (imbyungjun) [inline backend/src/main/kotlin/org/dontstw/eventfactory/domain/Member.kt:6]: Member... evar....
and... variable name... evar tuna...

Comment (hahava) [reply backend/src/main/kotlin/org/dontstw/eventfactory/domain/Member.kt:6]: Ok I will fix it ..

File: backend/src/main/kotlin/org/dontstw/eventfactory/domain/MemberRole.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package org.dontstw.eventfactory.domain
+
+data class MemberRole(
+        var roleName: String
+)
```
Comment (yonn-kim) [inline backend/src/main/kotlin/org/dontstw/eventfactory/domain/MemberRole.kt:5]: add empty line

File: backend/src/main/kotlin/org/dontstw/eventfactory/config/SecurityConfig.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        http!!.formLogin()
+        http!!.csrf().disable()
+    }
+
+    //TODO: It use memory auth but Should be replaced using jdbc.
+    override fun configure(auth: AuthenticationManagerBuilder?) {
+        // {noop} is meaning password not encoded
+        auth!!.inMemoryAuthentication().withUser("admin").password("{noop}1234").roles("ADMIN")
+    }
+}
```
Comment (yonn-kim) [inline backend/src/main/kotlin/org/dontstw/eventfactory/config/SecurityConfig.kt:38]: add empty line

**Pull request merged**